openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -532,6 +535,15 @@ def has_ended(self):
         """
         return course_metadata_utils.has_course_ended(self.end)
 
+    def enrollment_has_ended(self):
```

Comment:
```suggestion
    def has_enrollment_ended(self):
```
However, we might want to call it [is_enrollment_closed](https://github.com/openedx/edx-platform/blob/289e682b8f5fd4770aff5fe9147c365a5074a43e/common/djangoapps/student/models.py#L1376-L1383). Please see the comment below for an explanation.

Reply:
Changed it to `is_enrollment_open` as that more accurately captures what we are checking here